src/core/service.c

```diff
@@ -2577,7 +2579,7 @@ static void service_enter_reload(Service *s) {
                 r = service_spawn(s,
                                   s->control_command,
                                   s->timeout_start_usec,
-                                  EXEC_APPLY_SANDBOXING|EXEC_APPLY_CHROOT|EXEC_IS_CONTROL|EXEC_CONTROL_CGROUP,
+                                  EXEC_IS_CONTROL|EXEC_CONTROL_CGROUP,
```

Comment:
Here also.

Reply:
ExecReload= should signal the main process instead of accessing things like credentials too.

Reply:
Uh, we set that in `service_run_next_control`...